explanation/rendering-graphics.md

```diff
@@ -0,0 +1,38 @@
+(exp-rendering-graphics)=
+# Rendering graphical output
+
+Anbox Cloud can adapt to your rendering needs by adjusting its behaviour depending on the availability of a GPU and the presence of the `--enable-graphics` flag when you launch an instance.
+
+    amc launch --enable-graphics my-application
+
+The use of `--enable-graphics` flag when you launch an instance decides whether Anbox Cloud renders a visual output.
+
+When this flag is set, if there is a GPU available, Anbox Cloud renders visual output using the GPU. If there is no GPU available, it uses software rendering to render the visual output.
+
+However, if you do not use the `--enable-graphics` flag when launching an instance, Anbox Cloud does not render a visual output.
+
+Now your environment may or may not include a GPU. If Anbox Cloud detects a GPU device during deployment, it configures the cluster to use the GPU. AMS configures each LXD instance to pass through a GPU device from the host. Currently, all GPUs that are available to a machine are passed to every instance that owns a GPU slot. For NVIDIA GPUs, LXD uses the [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the instance.
+
+```{important}
+Anbox Cloud currently does not support GPU provisioning for virtual machines. This feature is planned for a future release.
+```
+
+To use container instances with a GPU,
+
+- Configure the number of available GPU slots on the node. This decides how many instances can run on a given node because GPUs have limited sharing capacity amongst multiple instances. See {ref}`sec-gpu-slots` for detailed information.
+- Check the list of supported GPUs ({ref}`sec-supported-gpus`) to see if Anbox Cloud includes a driver for your GPU device. If a GPU driver is available inside the instance, there are no further differences in how to use it in comparison to a regular environment. If no GPU driver is available, you must provide it through an {ref}`addon <exp-addons>`.
+- Launch your instance with the `--enable-graphics` flag.
+
+If your environment does not have a GPU, Anbox Cloud uses software or headless rendering.
```

Comment:
What do you mean by "headless rendering"? Headless means there is no output attached but it is not something which can replace "software rendering".

Reply:
Ok, so they are not interchangeable. Then, let's talk only about software rendering becauses that's what we use consistenly across our docs.